Description: turn off warnings as errors flag
Author: Scott Howard <showard@debian.org>
--- a/Makefile
+++ b/Makefile
@@ -85,13 +85,11 @@ endif
 ifeq ($(OS),Linux)
 COMMON_SRCS+=PosixSerialPort.cpp LinuxPortFactory.cpp
 COMMON_LIBS=-Wl,--as-needed
-COMMON_CXXFLAGS=-std=c++11
+COMMON_CXXFLAGS=-std=c++11 `dpkg-buildflags --get CXXFLAGS` `dpkg-buildflags --get CPPFLAGS`
 WX_LIBS+=-lX11
 
 MACHINE:=$(shell uname -m)
 
-install: strip
-	tar cvzf $(BINDIR)/bossa-$(MACHINE)-$(VERSION).tgz -C $(BINDIR) bossa$(EXE) bossac$(EXE) bossash$(EXE)
 endif
 
 #
@@ -100,7 +98,7 @@ endif
 ifeq ($(OS),Darwin)
 COMMON_SRCS+=PosixSerialPort.cpp OSXPortFactory.cpp
 COMMON_CXXFLAGS=-arch x86_64 -mmacosx-version-min=10.9
-COMMON_LDFLAGS=-arch x86_64 -mmacosx-version-min=10.9
+COMMON_LDFLAGS=-arch x86_64 `dpkg-buildflags --get LDFLAGS`
 APP=BOSSA.app
 DMG=bossa-$(VERSION).dmg
 VOLUME=BOSSA
